web/themes/custom/pineapple/components/_comments-field.twig

```diff
@@ -0,0 +1,45 @@
+{#
+/**
+ * @file
+ * Theme override for comment fields.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional title output populated by modules, intended to
+ *   be displayed after the main title tag that appears in the template.
+ * - comments: List of comments rendered through comment.html.twig.
+ * - comment_form: The 'Add new comment' form.
+ * - comment_display_mode: Is the comments are threaded.
+ * - comment_type: The comment type bundle ID for the comment field.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ * @see comment_preprocess_field()
+ */
+#}
+<section class="comments-wrapper">
+  {% if comment_form %}
+    <div class="comment-form-container">
+      <h2 class="subheading">{{ 'Leave a Comment'|t }}</h2>
+      {{ comment_form }}
+    </div>
+  {% endif %}
+
+  {% if comments and not label_hidden %}
+    <h2 class="subheading comments-title">Comments</h2>
+    {% if not comment_form %}
+      <div class="forbidden-comments">
+        <a href="/user/login">Log In</a><p> or</p><a href="/user/register"> register</a><p> to post comments.</p>
+      </div>
+    {% endif %}
+  {% endif %}
```

Comment:
Currently if you're logged out and on a recipe or article without any comments there's zero indication that you can add comments, this logic will need reworking so that there's guidance to point people to the possibility of adding comments. Maybe if you're logged out there can be some message that says something along the lines of 'There's no comments on this content yet.

The inset the Log in or register to post comments message/link.'

Reply:
Done